src/main/java/org/pentaho/vfs/ui/VfsFileChooserDialog.java

```diff
@@ -245,6 +247,7 @@ public void activate() {
   }
 
   private void selectCustomUI() {
+    firstTime = customUIPicker.getSelectionIndex() != 0;
```

Comment:
Doesn't index 0 of the customUIPicker correspond to "Local"? Not sure I understand why the variable is called "firstTime"